### PR TITLE
fix financial aid calculator imports

### LINKED ISSUE #1558: Cannot input income into cost calculator
When I type an number into the "Income (yearly)" field on the cost calculator, the Javascript console pops up an error:
```
FinancialAidCalculator.js:49 Uncaught TypeError: (0 , _profile.sanitizeNumberString) is not a function
```

When the field loses focus, the number I've typed in disappears. This makes the cost calculator unusable.

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #1558 

#### What's this PR do?

fixes a lil bug with the financial aid calculator. I introduced it when I merged https://github.com/mitodl/micromasters/pull/1534

#### How should this be manually tested?

Make sure you can add a new financial aid object to your user with no problems (in the UI).

### CODE REVIEW

**Comment by singingwolfboy** [comment]
Can we get an automated test for this, please?

**Comment by aliceriot** [comment]
yeah I was think we might want to get the fix out first - what do you think, should I add a test to this PR or do it in a follow-up?

**Comment by noisecapella** [comment]
@singingwolfboy can you assign yourself to a PR if you're reviewing it?

**Comment by aliceriot** [comment]
tests! @singingwolfboy @noisecapella which of you is reviewing?

in any case, things are ready to be looked at again.

**Comment by noisecapella** [comment]
@singingwolfboy is heading out so I'll take over

**Comment by aliceriot** [comment]
word

**Comment by noisecapella** [comment]
I'm getting this warning when running the tests, can you look into it?

      FinancialAidCalculator
        ✓ should let you open and close the financial aid calculator (562ms)
        ✓ should let you skip and pay full price (817ms)
        ✓ should let you enter your income (402ms)
        ✓ should let you enter your preferred currency (520ms)
    Warning: TextField is changing a controlled input of type text to be uncontrolled. Input elements should not switch from controlled to uncontrolled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
        ✓ should let you submit a financial aid request (882ms)

**Comment by aliceriot** [comment]
yea I saw that but I'm not sure what's up, I'm looking into it now

**Comment by aliceriot** [comment]
fixed!

File: static/js/reducers/financial_aid.js [hunk 1]
```diff
@@ -49,7 +51,7 @@ export const financialAid = (state: FinancialAidState = INITIAL_FINANCIAL_AID_ST
   case START_CALCULATOR_EDIT:
     return { ...FINANCIAL_AID_EDIT, programId: action.payload };
   case CLEAR_CALCULATOR_EDIT:
-    return INITIAL_FINANCIAL_AID_STATE;
+    return _.cloneDeep(FINANCIAL_AID_EDIT);
```
Comment (noisecapella) [inline static/js/reducers/financial_aid.js]: actions which are dispatched shouldn't be making direct modifications to the state so it seems like we don't need to clone this here

**Comment by noisecapella** [comment]
Functionality works great! There are a few uncovered cases when I look at the coverage report, I'll make a separate issue for those

**Comment by aliceriot** [comment]
updated!

**Comment by aliceriot** [comment]
It looks like some python tests are failing, not sure what's up

**Comment by noisecapella** [comment]
Can you rebase and force push? We should have the problems fixed finally

**Pull request merged**